src/armature/Model.ts

```diff
@@ -0,0 +1,66 @@
+import { mat3, mat4 } from 'gl-matrix';
+
+import { Node } from './Node';
+import { NodeRenderObject } from './NodeRenderObject';
+
+type RenderInfo = { currentMatrix: mat4; currentNormalMatrix: mat3; objects: NodeRenderObject };
```

Comment:
Huge nit, but could we make this multiple lines?

Reply:
autoformatter does this when there are no comments but there should really be comments anyway, I'll add those